core/arch/arm/kernel/proc_a32.S

```diff
@@ -39,6 +40,7 @@
 
 /* void cpu_spin_lock(lock address) - lock mutex */
 FUNC cpu_spin_lock , :
+UNWIND(	.fnstart)
```

Comment:
I think it's good to be explicit about the unwind directives. As I've understood it `.fnstart` and `.fnend` defines a range that usually matches start and end of a function, but it doesn't have to be that way.

I'd rather keep it as it is, but I don't have a strong opinion on it.